apps/web/src/app/(private)/_data-access/index.ts

```diff
@@ -0,0 +1,38 @@
+import { cache } from 'react'
+
+import {
+  findProject as fP,
+  type FindProjectProps,
+  findCommit as fC,
+  getFirstProject as gFP,
+  type FindCommitProps,
+} from '@latitude-data/core'
```

Comment:
Avoid using acronyms, as they may be confusing. Variable names are free ;)
Instead, try using descriptive variable names like ⁠`findProjectFn`.